app/src/main/java/com/android/sample/model/recipe/RecipesViewModel.kt

```diff
@@ -25,14 +39,24 @@ import kotlinx.coroutines.launch
  *
  * @property repository The repository used to fetch recipe data.
  */
-class RecipesViewModel(private val repository: RecipesRepository) :
-    ViewModel(), FilterPageViewModel, RecipeOverviewViewModel {
+class RecipesViewModel(
+    private val repository: RecipesRepository,
+    private val imageDownload: ImageDownload
+) : ViewModel(), FilterPageViewModel, RecipeOverviewViewModel {
 
   // StateFlow to monitor the list of recipes
   private val _recipes = MutableStateFlow<List<Recipe>>(emptyList())
   val recipes: StateFlow<List<Recipe>>
     get() = _recipes
 
+  private val _recipesTs = MutableStateFlow<List<Recipe>>(emptyList())
+  val recipesTs: StateFlow<List<Recipe>>
+    get() = _recipesTs
+
+  private val _recipesDl = MutableStateFlow<List<Recipe>>(emptyList())
+  val recipesDl: StateFlow<List<Recipe>>
+    get() = _recipesDl
+
```

Comment:
Hey could you give some more descriptive names to these flows ☺️

Reply:
Done!